app/models/mixins/compliance_mixin.rb

```diff
@@ -37,4 +40,14 @@ def compliance_policies
     _, plist = MiqPolicy.get_policies_for_target(self, "compliance", "#{target_class}_compliance_check")
     plist
   end
+
+  def last_compliance_conditions
```

Comment:
is there a reason why we need to resort to ruby and including 3 levels of classes here?

think @NickLaMuro already optimized `last_compliance` into a `has_one`, so we should be able to add a `:through` to get `last_compliance_conditions` and `last_compliance_condition_expressions`